### PR TITLE
bump scala versions

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What does this change?

Bumps slf4j 1.7.25 => 1.7.36
Bumps AWS libs 1.11 => 1.12
Bumps quartz-scheduler 2.2.3 => 2.3.2

[Bump jackson](https://github.com/orgs/playframework/discussions/11222)

No-op, this is for security vulnerabilities.

### CODE REVIEW

**Review by mxdvl — APPROVED** [review]
“what are you doing at work?”, says he

File: project/Dependencies.scala [hunk 1]
```diff
@@ -47,7 +47,7 @@ object Dependencies {
   val playGoogleAuth = "com.gu.play-googleauth" %% "play-v28" % "2.1.1"
   val playSecretRotation = "com.gu.play-secret-rotation" %% "play-v28" % "0.18"
   val playSecretRotationAwsSdk = "com.gu.play-secret-rotation" %% "aws-parameterstore-sdk-v1" % "0.18"
-  val quartzScheduler = "org.quartz-scheduler" % "quartz" % "2.2.3"
+  val quartzScheduler = "org.quartz-scheduler" % "quartz" % "2.3.2"
```
Comment (ioannakok) [inline project/Dependencies.scala:50]: Just double checking, isn't this the quartz version that broke admin app the other day? 

revert PR here: https://github.com/guardian/frontend/pull/24909/files

Comment (jamesgorrie) [reply project/Dependencies.scala:50]: Yup - I've tested admin and it seems to be working...

Comment (jamesgorrie) [reply project/Dependencies.scala:50]: Saying this - it's true that the admin app isn't starting up when deploying through riff-raff.

Comment (jamesgorrie) [reply project/Dependencies.scala:50]: This is why it wasn't starting:
https://github.com/guardian/frontend/pull/24962/commits/478fc2c0df68b41264465e0f8eb2b03628d75484

Basically the old version accepted the invalid syntax, there's [an issue](https://github.com/quartz-scheduler/quartz/issues/729) with others having the same problem.

[We should write some tests](https://github.com/guardian/frontend/issues/24964).

**Review by ioannakok — APPROVED** [review]

**Pull request merged**